cypress/e2e/awx/resources/workflowVisualizer.cy.ts

```diff
@@ -166,54 +141,274 @@ describe('Workflow Job templates visualizer', () => {
         cy.clickButton('Save');
         cy.get('[data-cy="alert-toaster"]').should('be.visible');
-        cy.get('[data-cy="wf-vzr-name"]').should('have.text', `${workflowJobTemplate.name}`);
         cy.get('button[data-cy="workflow-visualizer-toolbar-close"]').click();
         cy.verifyPageTitle(`${workflowJobTemplate.name}`);
       });
-    // Clean up
-    cleanup();
   });
   it('Remove all nodes using the kebab menu of the visualizer toolbar, save and delete the template', function () {
-    cy.createAwxWorkflowJobTemplate({
-      organization: (this.globalOrganization as Organization).id,
-      inventory: inventory.id,
-    }).then((workflowJobTemplate) => {
-      cy.createAwxWorkflowVisualizerProjectNode(
-        workflowJobTemplate,
-        this.globalProject as Project
-      ).then((projectNode) => {
-        cy.createAwxOrganization().then((org) => {
-          organization = org;
-          cy.createAwxProject({ organization: organization.id }).then((p) => {
-            project = p;
-            cy.createAwxInventorySource(inventory, project).then((invSrc) => {
-              inventorySource = invSrc;
-              cy.createAwxWorkflowVisualizerInventorySourceNode(
-                workflowJobTemplate,
-                inventorySource
-              ).then((inventorySourceNode) => {
-                cy.createAwxWorkflowVisualizerManagementNode(workflowJobTemplate, 1).then(
-                  (managementNode) => {
-                    cy.createWorkflowJTSuccessNodeLink(projectNode, inventorySourceNode);
-                    cy.createWorkflowJTAlwaysNodeLink(inventorySourceNode, managementNode);
-                  }
-                );
-              });
-            });
-          });
-        });
-      });
+    cy.createAwxWorkflowVisualizerProjectNode(workflowJobTemplate, project).then((projectNode) => {
+      cy.createAwxWorkflowVisualizerInventorySourceNode(workflowJobTemplate, inventorySource).then(
+        (inventorySourceNode) => {
+          cy.createAwxWorkflowVisualizerManagementNode(workflowJobTemplate, 1).then(
+            (managementNode) => {
+              cy.createWorkflowJTSuccessNodeLink(projectNode, inventorySourceNode);
+              cy.createWorkflowJTAlwaysNodeLink(inventorySourceNode, managementNode);
+            }
+          );
+        }
+      );
       cy.visit(`/templates/workflow_job_template/${workflowJobTemplate?.id}/visualizer`);
-      cy.get('[data-cy="wf-vzr-title"]')
-        .should('contain', 'Workflow Visualizer')
-        .should('be.visible');
       cy.get('[data-cy="wf-vzr-name"]')
         .should('contain', `${workflowJobTemplate.name}`)
         .should('be.visible');
-      //remove-node, add-node-and-link, add-link, add-link, edit-node
       cy.removeAllNodesFromVisualizerToolbar();
       cy.contains('button', 'Save').click();
       cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate);
     });
```

Comment:
Can we add an assertion after line 165 (215) that asserts that the nodes have been deleted? And since the test indicates that the user will delete the template, can we perform the deletion in the UI instead of through an HTTP request? either that or remove the piece of the test description that indicates that the test will be deleted?